test/unit/auditor_test.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+let chai = require("chai");
+let sinon = require("sinon");
+let sinonChai = require("sinon-chai");
+let expect = chai.expect;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'expect' is defined but never used.